### PR TITLE
Solves issue #321

### PR DESCRIPTION
**Pull request by Aishh2k**
Corrected information , links and image added for the section PSlab Android App
### **Before:**
screenshot 60

### **After:**
screenshot 61

### **Preview link:**
https://aishh2k.github.io/aishh.labs/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -552,14 +552,14 @@ <h1 class="text-white">PSLab Hardware</h1>
...
-                          <a href="https://www.facebook.com/eventyay/" target="_self" title="Facebook Profile" class="fa fa-facebook"></a>
-                          <a href="https://twitter.com/eventyay" target="_self" title="Twitter Profile" class="fa fa-twitter"></a>
+                          <a href="https://github.com/fossasia/pslab-android/blob/apk/app-master-release.apk" target="_self" title="Test App" class="fa fa-android"></a>
+                          <a href="https://groups.google.com/forum/#!forum/pslab-fossasia" target="_self"  title="Mailing List" class="fa fa-envelope"></a>
+                          <a href="https://gitter.im/fossasia/pslab" target="_self" title="Chat Channel" class="fa fa-comments"></a>
+                          <a href="https://github.com/fossasia/pslab-android" target="_self" title="Contribute" class="fa fa-github"></a>
+                          <a href="https://www.facebook.com/pslabio/" target="_self" title="Facebook Profile" class="fa fa-facebook"></a>
+                          <a href="https://twitter.com/pslabio" target="_self" title="Twitter Profile" class="fa fa-twitter"></a>
                     		  </div>
```
Comment (plxity) [inline index.html]: Intend it properly.

Comment (Aishh2k) [reply index.html]: okay!

**Comment by Aishh2k** [comment]
> Thanks, please change the Alt tag to include the info on PSLab and add icons and links for the Playstore and Fdroid.

@mariobehling  Done! :)

**Comment by Aishh2k** [comment]
> Please provide a better and smaller picture. Simply providing a screen shot from the website including the Windows panel is not suitable for a FOSS website. Thank you.

I have made the necessary changes! Sorry for the inconvenience .

**Comment by mariobehling** [comment]
Please see to it, when you have time:

> Please provide a better and smaller picture. Simply providing a screen shot from the website including the Windows panel is not suitable for a FOSS website. Thank you.

**Comment by mariobehling** [comment]
As the picture in this PR is still a Windows screenshot and no further follow up was made, I am closing this PR due to inactivity.

**Pull request closed without merging**